components/automate-chef-io/content/docs/desktop.md

```diff
@@ -0,0 +1,41 @@
++++
+title = "Desktop"
+description = "Using the Desktop View"
+date = 2018-03-26T16:01:47-07:00
+draft = true
+bref = ""
+toc = true
++++
+
+## Overview
+The _Desktop_ page displays information about all desktops connected to Chef Automate. It can be found via the top nav of your Automate instance, at `automate-url/desktop`
+Desktops appear in this view after a Chef Infra Client run has executed.
+
+### Daily Check In
+Shows a count of desktops that have and are reporting in to Automate. Desktops that have not checked in in the last twenty-four hours will be counted as `unknown`.
+Selecting a status filter opens the list view of the desktops (Insight) with the selected status.
+
+### Check In History
+Shows the history of checked-in and unknown desktop counts over time. Timescale can be adjusted using the timescale dropdown menu.
+
+### Top Errors
+Shows the most common errors that have occurred across all desktops over the last twenty-four hours, and a count of the machines that incurred the error.
+Selecting an error opens the list view of the desktops (Insight) with the error to enable further investigation.
+
+### Unknown Duration
```

Comment:
@vjeffrey is this now "Time since Last Check-in"?

Reply:
ya!